packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -121,20 +133,27 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
                 </div>
               </div>
 
-              <div className={classNames.code}>{isCodeVisible && <CodeSnippet language="tsx">{code}</CodeSnippet>}</div>
-
-              {activeCustomizations ? (
-                <CustomizerContext.Provider value={{ customizations: { settings: {}, scopedSettings: {} } }}>
-                  <Customizer {...activeCustomizations}>
-                    <ThemeProvider scheme={_schemes[schemeIndex]}>
-                      <Stack styles={regionStyles}>{exampleCardContent}</Stack>
-                    </ThemeProvider>
-                  </Customizer>
-                </CustomizerContext.Provider>
-              ) : (
-                exampleCardContent
+              {isCodeVisible && !this.canRenderLiveEditor && (
+                <div className={classNames.code}>
+                  <CodeSnippet language="tsx">{code}</CodeSnippet>
+                </div>
               )}
 
+              {isCodeVisible && this.canRenderLiveEditor && editor}
+
+              {(!isCodeVisible || !this.canRenderLiveEditor) &&
+                (activeCustomizations ? (
+                  <CustomizerContext.Provider value={{ customizations: { settings: {}, scopedSettings: {} } }}>
+                    <Customizer {...activeCustomizations}>
+                      <ThemeProvider scheme={_schemes[schemeIndex]}>
+                        <Stack styles={regionStyles}>{exampleCardContent}</Stack>
```

Comment:
The customizations need to be applied to the EditorPreview as well, so you'll probably need to do something like this: 
```suggestion
                        <Stack styles={regionStyles}>{this.canRenderLiveEditor ? <EditorPreview ... /> : exampleCardContent}</Stack>
```